src/vdom/component.js

```diff
@@ -7,11 +7,14 @@ import { diff, mounts, diffLevel, flushMounts, recollectNodeTree, removeChildren
 import { createComponent, collectComponent } from './component-recycler';
 import { removeNode } from '../dom/index';
 
-/** Set a component's `props` (generally derived from JSX attributes).
- *	@param {Object} props
- *	@param {Object} [opts]
- *	@param {boolean} [opts.renderSync=false]	If `true` and {@link options.syncComponentUpdates} is `true`, triggers synchronous rendering.
- *	@param {boolean} [opts.render=true]			If `false`, no render will be triggered.
+/**
+ * Set a component's `props` (generally derived from JSX attributes).
+ * @param {Object} props
+ * @param {Object|number} [opts] If opts is a `number` it is equal to the render
```

Comment:
I don't think the `opts` parameter of `setComponentProps` is used as an options object anymore. It appears to just be the render mode. At least I didn't see any usages of `opts` as an object.

Reply:
Good catch!